src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -109,7 +122,7 @@ function PostEditor({
     } else {
       const cohort = canSelectCohort
         // null stands for no cohort restriction ("All learners" option)
-        ? (values.cohort ?? null)
+        ? (values.cohort || null)
```

Comment:
I saw that there was a commit that replaced `??` with `||`, and I didn't pay much attention to it. I understand that here it doesn't really make any difference, since `values.cohort` is either a non empty string or a "nullish" value, but `??` semantically hints that we are checking for "nullish" values.
Can you help me understand what is the purpose of this change?

Reply:
The purpose of this change is that in some cases this is causing the tests to fail because the tests run on node 12, and node 12 doesn't support `??` yet. I don't know why this is happening, but tests consistently fail in this case. 

I've replaced situations where we're dealing with non booleans from ?? to || since it shouldn't make a different.